modules/apps/commerce/commerce-product-content-search-web/src/main/java/com/liferay/commerce/product/content/search/web/internal/portlet/shared/search/CPSearchResultsPortletSharedSearchContributor.java

```diff
@@ -70,11 +71,14 @@ public void contribute(
 			SearchRequestBuilder searchRequestBuilder =
 				portletSharedSearchSettings.getSearchRequestBuilder();
 
-			Optional<String> paginationStartParameterNameOptional =
-				portletSharedSearchSettings.getPaginationStartParameterName();
+			if (searchRequestBuilder == null) {
```

Comment:
@julschong To be consistent with the original logic, you should perform null check on `portletSharedSearchSettings.getPaginationStartParameterName()`.